extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -190,25 +209,39 @@ export default class JupyterServerInstallation {
 
 	private configurePackagePaths(): void {
 		//Python source path up to bundle version
-		let pythonSourcePath = path.join(this._pythonInstallationPath, constants.pythonBundleVersion);
+		let pythonSourcePath = this._usingExistingPython
+			? this._pythonInstallationPath
+			: path.join(this._pythonInstallationPath, constants.pythonBundleVersion);
 
 		this._pythonPackageDir = path.join(pythonSourcePath, 'offlinePackages');
 
 		// Update python paths and properties to reference user's local python.
 		let pythonBinPathSuffix = process.platform === constants.winPlatform ? '' : 'bin';
 
-		this._pythonExecutable = JupyterServerInstallation.getPythonExePath(this._pythonInstallationPath);
+		this._pythonExecutable = JupyterServerInstallation.getPythonExePath(this._pythonInstallationPath, this._usingExistingPython);
 		this.pythonBinPath = path.join(pythonSourcePath, pythonBinPathSuffix);
 
+		this._usingConda = this.checkCondaExists();
+
 		// Store paths to python libraries required to run jupyter.
 		this.pythonEnvVarPath = process.env['PATH'];
 
 		let delimiter = path.delimiter;
+		this.pythonEnvVarPath = this.pythonBinPath + delimiter + this.pythonEnvVarPath;
 		if (process.platform === constants.winPlatform) {
 			let pythonScriptsPath = path.join(pythonSourcePath, 'Scripts');
 			this.pythonEnvVarPath = pythonScriptsPath + delimiter + this.pythonEnvVarPath;
+
+			if (this._usingConda) {
+				this.pythonEnvVarPath = [
+					path.join(pythonSourcePath, 'Library', 'mingw-w64', 'bin'),
```

Comment:
What about for non-Windows platforms? Why is this Windows specific?

Reply:
Windows Anaconda has a bunch of dependencies all over the place that are required in the PATH. The Linux/Mac versions just use the bin folder, from what I remember.

Reply:
How does the VSCode python extension handle this?

Reply:
They have a very different architecture for handling different types of Python. I'm still looking through it.

Reply:
Seems like the Python extension uses whatever PATH is already defined. I get Not Found errors when trying to run conda right away via the Python extension. In general, Windows Anaconda requires terminal commands to be run through a provided Anaconda Prompt, which takes care of all the environment setup. These new PATH locations were pulled from the PATH variable in the Anaconda Prompt.